content/page/faq.md

```diff
@@ -0,0 +1,20 @@
+![Installing LinuxBoot on an Intel S2600 mainboard](images/s2600-flash.jpg)
```

Comment:
overall this discussion is going to be confusing to people but I'll try to add some text. You know so much about this area. We all forget that things that are obvious and easy are unknown to most people.